### ISSUE TITLE
Missing reserved words

### ISSUE THREAD

**Issue by antoinecarme** [Teradata/sqlalchemy-teradata#41]
Hi,

Thanks a lot for this package. I am using it with the free vmware image of teradata. It works like a charm.

I am generating a lot of SQL code from machine learning models for various databases. you can look at https://github.com/antoinecarme/sklearn2sql-demo/tree/master/VeryLargeModelsSupport_temp_tables/DecisionTreeClassifier/iris for an example.

I am creating this issue to report some missing reserved words (using the last version, searched the list of existing issues). The following words cannot be used properly as identifiers of columns in teradata (have to be quoted) :

```Python
["index" , "threshold" , "class", "quantile"]
```

By adding this line in base.py I was able to make things work :

```Python
ReservedWords = ReservedWords | set (["index" , "threshold" , "class", "quantile"])
```

I attach here a sample script for testing:

```Python
import sqlalchemy as sa
engine1 = sa.create_engine(lDSN1, echo=True)
conn = engine1.connect()

from sqlalchemy import *

lColumnName = "index" # also "class" or "threshold" , "quantile"
lMeta = MetaData()
lTestTable = Table("tmp_123_xasdfsefc", lMeta, Column(lColumnName, Float , primary_key=True));
        
print(CreateTable(lTestTable))

conn.execute(DropTable(lTestTable))
conn.execute(CreateTable(lTestTable))

```

Probably, the list of reserved words is even longer (according to teradata 16.2 manual).

Again, thanks a lot for the good work.

Regards

Antoine

**Comment by sandan** [comment]
@antoinecarme Thanks! Would you like to open a PR to get credit for your work?

**Comment by antoinecarme** [comment]
@sandan,

Thanks a lot for your feedback. This is a rather small issue. All the credit goes to the excellent team that provided this driver. What I diid was just a work-around not a real fix.

My knowledge of teradata is limited. I think a real Teradata expert would better handle this by adding all possible reserved words, for example.

**Comment by sandan** [comment]
Fixed by 457752d001297ae41e982fd584b2e1791d36a6d2
